File: PublicTransport/PublicTransport.Api/Models/Ticket.cs
using System;

namespace PublicTransport.Api.Models
{
    public class Ticket
    {
        public int Id { get; set; }
        public DateTime DateOfIssue { get; set; }
        public string TicketType { get; set; }
        public bool IsValid { get; set; }
        public User User { get; set; }
        public PricelistItem PriceInfo { get; set; }
    }
}

File: PublicTransport-Spa/src/app/_models/ticket.ts
import { PricelistItem } from './pricelistItem';
import { User } from './user';

export interface Ticket {
    id: number;
    dateOfIssue: Date;
    ticketType: string;
    isValid: boolean;
    user: User;
    priceInfo: PricelistItem;
    checkResult?: string;
}
